assets/src/edit-story/app/uploader/utils/error.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Helper function to get create a js error.
+ *
+ * @param {string} name Error name.
+ * @param {string} fileName File name.
+ * @param {string} message Message in error.
+ * @return {Error} Error Object.
+ */
+function createError(name, fileName, message) {
+  const validError = new Error();
+
+  validError.name = name;
+  validError.file = fileName;
+  validError.isUserError = true;
```

Comment:
What effect does this have?

Reply:
This is copied from where it was before. I haven't touched it.

Reply:
Seems to have been added by @ndev1991 here https://github.com/google/web-stories-wp/commit/0c1c2a48c72fbf898c1bcbf90fc4adfb1e2a90db

Can you give us some idea what this does and why it was added.

Reply:
It was added to have the correct information of each error type. Basically for the information of snackbar error.

Reply:
This doesn't provide a lot of detail of why. It seems to be have added to tell the difference between a user generated error and a compiler error. But this is not a valid prop of a error object. Is that correct?

Reply:
I'm not sure the current upload multiple files logic, but basically there is snack bar error that shows file names failed to upload. It was added at that time to get all the correct file names that failed to upload.